Sources/PassKit/Models/ConcreteModels.swift

```diff
@@ -32,7 +32,7 @@ import Fluent
 final public class PKDevice: PassKitDevice {
     public static let schema = "devices"
 
-    @ID(key: "id")
+    @ID(custom: "id")
```

Comment:
It's recommended to use `.id` instead of `"id"` for `@ID` properties, whether custom or not.
```suggestion
    @ID(custom: .id)
```